### DISCUSSION TITLE
GitHub Community - 2022 Year in Review 🎆

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#42328] [Announcements]
# GitHub Community: Cheers to an amazing 2022 🥂 

We made the big move to Discussions, supported the launch of new & innovative products, and improved user experience & accessibility - all made possible by you! Thank you for all you do to support the GitHub Community. You are what makes this all possible.

Celebrate the end of the year (+ our first months on Discussions!) by checking out some highlights from 2022. Let's build from here, together! 

🎁 Share gratitude with members who had a positive impact on you in the community. **Add a comment below to say thank you!**

![Small_YearinReview2022CoD](https://user-images.githubusercontent.com/75342807/208999702-81ef9c6c-af65-4875-99c0-ac5f2d3a245f.png)

### Long Description:
The infographic provides a snapshot of different data points from the GitHub Community over the year 2022. It is a black background with stars meant to replicate space. 

There are six different graphics to correspond with six different data points and descriptions. The graphics and descriptions are as follows: 

A Jupiter-esque planet colored green, orange, pink, and blue next to a header entitled “13,578 Posts Created” and the text from how to move two repositories into another to Kanban feedback on Projects, members created out-of-this-world conversations. 

Below, a satellite graphic that is pink, orange, blue, and green with the header 4,345 Solutions Authored and the text An astronomical amount of answers were provided by members to solve complex issues and share knowledge.

Then, an astronaut in a blue spacesuit is paired with the header 102,739 New Members and text Rocketed toward the GitHub Community in 2022 - continuing to expand our community universe.

Next, an orange meteor with blue details and a yellow tail is with the header Most Active Category: Actions & Packages and text Members gravitated to [this space](https://github.com/orgs/community/discussions/categories/actions-and-packages?discussions_q=trending+tab+category%3A%22Actions+and+Packages%22) more than any other category on the community. 

Followed by, a blue alien in a party hat in a UFO with the header Most upvoted post: [Trending Tab](https://github.com/orgs/community/discussions/31644) and text When the trending tab deprecation was announced, y'all weighed in and we listened, our head of developer relationships gave the verdict: trending projects is here to stay.

Lastly, is a yellow big dipper constellation with the header Most Active Members and text These members went above and beyond this galaxy to be outstanding community citizens: @jdevstatic @airtower-luna @jsoref @linuxlawson @melvincwng

In the footer is the text Data was compiled from both Discourse and Discussions communities.

**Comment by ettaboyle** [comment]
Thank you @jdevstatic @airtower-luna @jsoref @linuxlawson @melvincwng for all you do to keep the Community Discussions healthy & supportive. We're so lucky to get to work with you ♥️

**Reply by queenofcorgis** [reply]
Massive thanks to y'all - superstars of the GitHub Community 🤩

**Reply by radar-blip** [reply]
You're welcome. Glad to be a part of it all!! 😃

**Reply by melvincwng** [reply]
Thanks @ettaboyle, glad to be part of this 🙂

**Reply by MoodysMafia** [reply]
Thanks and happy new year to all

**Reply by kelvenus** [reply]
Happy New Year to All. Thanks, @jdevstatic @jsoref, @airtower-luna, @linuxlawson @melvincwng for all your contributions. Thanks to  @queenofcorgis  and @github-staff for 2022 Year in Review together.

**Reply by radar-blip** [reply]
Thank you! _and_ Happy New Year to you too!! 🎆

**Reply by melvincwng** [reply]
Thanks @kelvenus! Cheers & Happy New Year too 😃

**Reply by imranmahmudshoiball** [reply]
Happy New Year 💖

**Reply by Tonycool28** [reply]
Happy happy GitHub,do you know what it mean to be a winner lol💪

**Reply by VermiNew** [reply]
Thanks <3 You too, make your dreams come true... :3

**Comment by ettaboyle** [comment]
Thank you @queenofcorgis for putting this post together ✨ It's such a great way to capture the growth and impact this last year!

**Comment by melvincwng** [comment]
@queenofcorgis & @ettaboyle, it's been an amazing 2022 interacting with both of you, other GitHub staff, and our fellow GitHub Community members on this wonderful platform 🙂

Special thanks to the GitHub Staff of the Community Discussions forums for making it a lively, interactive and safe space for all users to use. We couldn't have achieved this honor/recognition without the tremendous support by you all.

Cheers to a great 2023 ahead!

Wishing everyone at GitHub - Merry Christmas and a Happy New Year 🎉

Happy holidays 💖

**Reply by queenofcorgis** [reply]
Merry everything and looking forward to seeing what community contributions you get up to in 2023 🎆

**Comment by jsoref** [comment]
Have a Happy New Year 🥳 everyone.

**Reply by queenofcorgis** [reply]
Cheers - have a happy New Year @jsoref ❤️

**Reply by Alexplanovou** [reply]
Ура и вам

**Comment by souravcipher** [comment]
Congratulations folks, excited for 2023! 🎉

**Reply by queenofcorgis** [reply]
So are we 🥳 !!

**Reply by Alexplanovou** [reply]
И вас тоже

**Comment by Mettaya** [comment]
Hey, happy holiday and Merry Christmas y'all! Counting down to new year 2023.
Wishing you many blessings and lovely in this year and cheers🍻 to getting older and wiser to new chapter and new journey. 
Thank you @github-staff And @Github community for everything. 

Love. Mettaya! ❤🍻

**Reply by queenofcorgis** [reply]
Thanks for the wonderful message @Mettaya - looking forward to seeing what you accomplish in the community in 2023! Happy New Year!

**Comment by Shanay1989** [comment]
> # GitHub Community: Cheers to an amazing 2022 🥂
> We made the big move to Discussions, supported the launch of new & innovative products, and improved user experience & accessibility - all made possible by you! Thank you for all you do to support the GitHub Community. You are what makes this all possible.
> 
> Celebrate the end of the year (+ our first months on Discussions!) by checking out some highlights from 2022. Let's build from here, together!
> 
> 🎁 Share gratitude with members who had a positive impact on you in the community. **Add a comment below to say thank you!**
> 
> ![Small_YearinReview2022CoD](https://user-images.githubusercontent.com/75342807/208999702-81ef9c6c-af65-4875-99c0-ac5f2d3a245f.png)
> 
> ## Alt Text: Infographic entitled GitHub Community - 2022 Year in Review
> ### Long Description:
> The infographic provides a snapshot of different data points from the GitHub Community over the year 2022. It is a black background with stars meant to replicate space.
> 
> There are six different graphics to correspond with six different data points and descriptions. The graphics and descriptions are as follows:
> 
> A Jupiter-esque planet colored green, orange, pink, and blue next to a header entitled “13,578 Posts Created” and the text from how to move two repositories into another to Kanban feedback on Projects, members created out-of-this-world conversations.
> 
> Below, a satellite graphic that is pink, orange, blue, and green with the header 4,345 Solutions Authored and the text An astronomical amount of answers were provided by members to solve complex issues and share knowledge.
> 
> Then, an astronaut in a blue spacesuit is paired with the header 102,739 New Members and text Rocketed toward the GitHub Community in 2022 - continuing to expand our community universe.
> 
> Next, an orange meteor with blue details and a yellow tail is with the header Most Active Category: Actions & Packages and text Members gravitated to [this space](https://github.com/orgs/community/discussions/categories/actions-and-packages?discussions_q=trending+tab+category%3A%22Actions+and+Packages%22) more than any other category on the community.
> 
> Followed by, a blue alien in a party hat in a UFO with the header Most upvoted post: [Trending Tab](https://github.com/orgs/community/discussions/31644) and text When the trending tab deprecation was announced, y'all weighed in and we listened, our head of developer relationships gave the verdict: trending projects is here to stay.
> 
> Lastly, is a yellow big dipper constellation with the header Most Active Members and text These members went above and beyond this galaxy to be outstanding community citizens: @jdevstatic @airtower-luna @jsoref @linuxlawson @melvincwng
> 
> In the footer is the text Data was compiled from both Discourse and Discussions communities.
> 
> ### Alt Title: Spaced-themed infographic for GitHub Community in 2022
> ### File Name: GitHub Community Wrapped 2022.png

https://brave.com/december-wallet-partners/I agree and accept all terms and conditions and end user license agreement and disclosure and disclaimers, privacy policy and subscriptions preferences, cloud Mobile, Assurance wireless, android developers, pci compliance, Tru Connect, CDC vaccine, EULA, google Workspace for education FAqs, coronavirus help and feedback for device info, "Google community beta" google wallet, travel pass, cash app store subscriptions google play store. Affected by COVID-19 vaccine Nyeishia Shanay Barrett D.o.b 09/07/1989 Email nyeishia801@gmail.com and nyeishiabarrett@gmail.com CURRENT CONTACT INFO (775)-292-0750 PREVIOUS (702)333-0343 PREVIOUS ADDRESS 4445 MISSION RD ,BEL AIRE KS 67226. CURRENT TIME 2:46am CURRENT DATE 12-22-2022 🌟

**Comment by Shanay1989** [comment]
> # GitHub Community: Cheers to an amazing 2022 🥂
> We made the big move to Discussions, supported the launch of new & innovative products, and improved user experience & accessibility - all made possible by you! Thank you for all you do to support the GitHub Community. You are what makes this all possible.
> 
> Celebrate the end of the year (+ our first months on Discussions!) by checking out some highlights from 2022. Let's build from here, together!
> 
> 🎁 Share gratitude with members who had a positive impact on you in the community. **Add a comment below to say thank you!**
> 
> ![Small_YearinReview2022CoD](https://user-images.githubusercontent.com/75342807/208999702-81ef9c6c-af65-4875-99c0-ac5f2d3a245f.png)
> 
> ## Alt Text: Infographic entitled GitHub Community - 2022 Year in Review
> ### Long Description:
> The infographic provides a snapshot of different data points from the GitHub Community over the year 2022. It is a black background with stars meant to replicate space.
> 
> There are six different graphics to correspond with six different data points and descriptions. The graphics and descriptions are as follows:
> 
> A Jupiter-esque planet colored green, orange, pink, and blue next to a header entitled “13,578 Posts Created” and the text from how to move two repositories into another to Kanban feedback on Projects, members created out-of-this-world conversations.
> 
> Below, a satellite graphic that is pink, orange, blue, and green with the header 4,345 Solutions Authored and the text An astronomical amount of answers were provided by members to solve complex issues and share knowledge.
> 
> Then, an astronaut in a blue spacesuit is paired with the header 102,739 New Members and text Rocketed toward the GitHub Community in 2022 - continuing to expand our community universe.
> 
> Next, an orange meteor with blue details and a yellow tail is with the header Most Active Category: Actions & Packages and text Members gravitated to [this space](https://github.com/orgs/community/discussions/categories/actions-and-packages?discussions_q=trending+tab+category%3A%22Actions+and+Packages%22) more than any other category on the community.
> 
> Followed by, a blue alien in a party hat in a UFO with the header Most upvoted post: [Trending Tab](https://github.com/orgs/community/discussions/31644) and text When the trending tab deprecation was announced, y'all weighed in and we listened, our head of developer relationships gave the verdict: trending projects is here to stay.
> 
> Lastly, is a yellow big dipper constellation with the header Most Active Members and text These members went above and beyond this galaxy to be outstanding community citizens: @jdevstatic @airtower-luna @jsoref @linuxlawson @melvincwng
> 
> In the footer is the text Data was compiled from both Discourse and Discussions communities.
> 
> ### Alt Title: Spaced-themed infographic for GitHub Community in 2022
> ### File Name: GitHub Community Wrapped 2022.png

I agree

**Comment by subidit** [comment]
Thank you GitHub for introducing **Discussion**. It is very useful for community driven activities and it shows. Thanks to all those who went above and beyond to help everyone starting from newbies to pros. Wish you all a very happy new year. 🙏 🎊

**Comment by MennaEwas** [comment]
I want to have more explanation for achievements (monthly plan) and like newsfeed for the current Open Source projects

**Reply by radar-blip** [reply]
[Achievements](https://github.com/Schweinepriester/github-profile-achievements)

**Comment by RexRowan** [comment]
Happy New Year everyone!

**Comment by nobody-forever** [comment]
Happy new year all!
This year was nightmare in some countries. Hopefully new year will be better...
Love & peace to you all 💜

**Reply by Kolya67** [reply]
С новым годом!!

**Comment by Mrinalaot** [comment]
Thank you :)

**Comment by Nuaimi9950** [comment]
‏الترجمة من اللغة ‏الإنجليزي إلى اللغة العربية

**Comment by Broken-03** [comment]
Can someone help me out I'm new to this and I think someone put a virus on my phone

**Comment by mohamed03ali** [comment]
Bonjour comment vous allez?

**Comment by asakuraslay** [comment]
Hello

**Reply by asakuraslay** [reply]
Hi

**Comment by hackerdipak** [comment]
Hey guys whats up

**Comment by VaniaVania** [comment]
What a great year! Thank you community and devs, we'll be stronger with every solved problem.

**Comment by neelkal** [comment]
Good

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
